code/core/src/manager/container/Menu.tsx

```diff
@@ -223,10 +204,28 @@ export const useMenu = (
       title: 'Collapse all',
       onClick: () => api.emit(STORIES_COLLAPSE_ALL),
       right: enableShortcuts ? <Shortcut keys={shortcutKeys.collapseAll} /> : null,
+      icon: <></>,
     }),
     [api, enableShortcuts, shortcutKeys]
   );
 
+  const documentation = useMemo(() => {
+    const docsUrl = api.getDocsUrl({ versioned: true, renderer: true });
+
+    return {
+      id: 'documentation',
+      title: 'Documentation',
+      closeOnClick: true,
+      href: docsUrl,
+      right: (
+        <Listbox.Icon>
+          <ShareAltIcon />
+        </Listbox.Icon>
+      ),
+      icon: <DocumentIcon />,
+    };
+  }, [api]);
```

Comment:
Can you define a return type so that it is clear why this object has its particular shape?